src/components/SettingsView/SettingsView.js

```diff
@@ -99,6 +100,17 @@ const SettingsView = ({
             }
             label="Display custom names for cows received from other players"
           />
+          <FormControlLabel
+            control={
+              <Switch
+                color="primary"
+                checked={useLegacyLevelingSystem}
+                onChange={() => console.log('todo: save change')}
+                name="use-legacy-leveling-system"
+              />
+            }
+            label="Use legacy leveling system (items sold)"
```

Comment:
What do you think about a small copy change to make this a bit clearer to unfamiliar players?

```suggestion
            label="Use legacy leveling system (experience is only gained by selling items)"
```